src/test/kotlin/uk/gov/justice/digital/hmpps/nomisvisitsmappingservice/casenotes/CaseNoteMappingResourceIntTest.kt

```diff
@@ -983,8 +983,8 @@ class CaseNoteMappingResourceIntTest : IntegrationTestBase() {
     }
 
     @Test
-    fun `get casenote mappings by migration id success`() {
-      (1L..2L).forEach {
+    fun `get casenote mapping count success`() {
+      (1L..140L).forEach {
```

Comment:
Rather than hardcoding 134 would it be better to make it configurable? Then you could run the test with 3 case notes instead of 140 making it simpler and quicker.

Reply:
good idea